### PR TITLE
Add Robomongo keyword so users can find it by searching 'robomongo'

### PR DESCRIPTION
**Pull request by danielstaleiny**
###### Motivation for this change

This should make it easier to find robomongo package under robo3t.

I was looking for robomongo and didn't find it. It is annoying to find that robomongo has different name as package and it is not searchable by robomongo. I suspect that I am not alone.

###### Things done

I just added robomongo keyword to description so you can find it when searching for it.

Adding word to string will not break anything.

### CODE REVIEW

File: pkgs/applications/misc/robo3t/default.nix [hunk 1]
```diff
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
 
   meta = {
     homepage = https://robomongo.org/;
-    description = "Query GUI for mongodb";
+    description = "Robomongo, Query GUI for mongodb";
```
Comment (erikarvstedt) [inline pkgs/applications/misc/robo3t/default.nix]: `description = "Query GUI for mongodb. Formerly called Robomongo.";`

**Comment by danielstaleiny** [comment]
I revisited the change and changed the name based on suggestion.

File: pkgs/applications/misc/robo3t/default.nix [hunk 2]
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
     name = "robo3t";
     exec = "robo3t";
     icon = icon;
-    comment = "Query GUI for mongodb";
+    comment = "Query GUI for mongodb. Formerly called Robomongo.";
```
Comment (erikarvstedt) [inline pkgs/applications/misc/robo3t/default.nix]: I don't think its necessary to repeat the alias here, it's only useful in `meta.description` for `nix search`.

**Review by dasJ — APPROVED** [review]

**Comment by dasJ** [comment]
Ah hold up. Package descriptions usually don't end with a dot

**Review by dasJ — APPROVED** [review]

**Review by emanueleperuffo — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**